applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -39,6 +41,7 @@ config BLE_ACL_PER_ADV_INT_MAX
 config BLE_ACL_EXT_ADV_INT_MIN
 	hex "Minimum extended advertising interval"
 	range 0x0030 0x0780
+	default 0x30 if TRANSPORT_BIS
```

Comment:
Could you please elaborate on these specific choices for BIS?

Reply:
This limitation must be mentioned here.

Reply:
Done